lib/epa.ex

```diff
@@ -22,19 +22,23 @@ defmodule EPA do
   end
 
   @doc ~S"""
-  Does a few things things:
+  Does a few things:
     1. Gets the ENV vars out of the environment
     1. Strips leading and trailing whitespaces from the values
     1. Checks that all values are non-nil, non-empty strings.
     1. Raises an exception on failure
 
-  Optionally takes an atom specifying the environment these vars are required in.
+  Optionally takes an atom or list of atoms specifying the environment these vars are required in.
   """
   @spec required([env_var], environment)
   :: true | no_return
-  def required(expected_vars, env) when is_atom(env) and is_list(expected_vars) do
-    if Mix.env == env do
+  def required(expected_vars, env) when is_list(expected_vars) do
```

Comment:
Can we do `is_list(expected_vars) and (is_list(env) or is_atom(env))`? Then this can be version `0.1.2` instead of a breaking change.

Reply:
👏 good call!